src/plus/github/githubGitProvider.ts

```diff
@@ -1246,6 +1249,17 @@ export class GitHubGitProvider implements GitProvider, Disposable {
 						current: true,
 					},
 				];
+
+				if (branch.upstream?.name != null) {
+					// Add the branch name (tip) to the upstream name entry in the downstreams map
+					let downstreams = downstreamMap.get(branch.upstream.name);
+					if (downstreams == null) {
+						downstreams = [];
+					}
+
+					downstreams.push(branch.name);
+					downstreamMap.set(branch.upstream.name, downstreams);
```

Comment:
Same here.

Reply:
Done in commit: [Optimizations](https://github.com/gitkraken/vscode-gitlens/pull/2552/commits/822bbc116c4740fb919c2a1af03d019b4a56e701)